ksqldb-functional-tests/src/test/resources/historical_plans/joins_-_stream-table_key-to-key_-_SR-enabled_key_format/6.2.0_1606950106050/topology

```diff
@@ -3,20 +3,17 @@ Topologies:
     Source: KSTREAM-SOURCE-0000000001 (topics: [t])
       --> KTABLE-SOURCE-0000000002
     Processor: KTABLE-SOURCE-0000000002 (stores: [])
-      --> KTABLE-MAPVALUES-0000000003
```

Comment:
Deleting and regenerating historic plans in this way is normally a major no-no but it was a conscious decision in this case as we have not released SR-enabled joins yet. This PR will be merged before 0.15 is cut, so these topology changes will not appear as a breaking change between releases.